composeApp/src/commonMain/kotlin/com/github/familyvault/services/ChatService.kt

```diff
@@ -164,6 +166,39 @@ class ChatService(
         }
     }
 
+    override suspend fun retrieveChatThreadManagers(threadId: String): List<String> {
+        val threadItem = privMxClient.retrieveThread(threadId)
+
+        return threadItem.managers
+    }
+
+    override suspend fun retrieveThreadPrivateMeta(threadId: String): ThreadPrivateMeta {
+        val threadItem = privMxClient.retrieveThread(threadId)
+
+        return threadItem.privateMeta
+    }
```

Comment:
To moim zdaniem można bardziej doprecyzować, zamiast rzucać po prostu `PrivateMeta`, gdzieś niżej w komentarzu napisałem.